pkg/reconciler/apiserversource/resources/receive_adapter.go

```diff
@@ -41,9 +42,9 @@ func MakeReceiveAdapter(args *ReceiveAdapterArgs) *v1.Deployment {
 	replicas := int32(1)
 	return &v1.Deployment{
 		ObjectMeta: metav1.ObjectMeta{
-			Namespace:    args.Source.Namespace,
-			GenerateName: fmt.Sprintf("apiserversource-%s-", args.Source.Name),
-			Labels:       args.Labels,
+			Namespace: args.Source.Namespace,
+			Name:      utils.GenerateFixedName(args.Source, fmt.Sprintf("apiserversource-%s", args.Source.Name)),
```

Comment:
should it be `apiserverimporter-%s`?

Reply:
I'd rather leave this name synced to the name of the CRD. If and when we change the name of the CRD, then we should change this name to match.